source/jormungandr/jormungandr/external_services/external_service.py

```diff
@@ -89,10 +85,10 @@ def get_response(self, arguments):
 
     @classmethod
     def _check_response(cls, response):
-        if not response:
+        if response is None:
             raise ExternalServiceError('impossible to access external service')
         if response.status_code == 503:
-            raise ExternalServiceUnavailable('forseti responded with 503')
+            raise ExternalServiceError('forseti responded with 503')
```

Comment:
Can you add the message returned by Forseti, No?

![image](https://user-images.githubusercontent.com/4006603/123668303-42c7ea00-d83b-11eb-9b14-8c7c588d4640.png)

Reply:
Infact we dont 't need this exception. It is already handled in the following lines.